docker.go

```diff
@@ -99,5 +106,27 @@ func loadDockerImage(cli *client.Client, pathToImage string) (string, error) {
 
 	b, err := ioutil.ReadAll(ret.Body)
 
-	return string(b[:]), nil
+	// {"stream":"Loaded image: sles12/mariadb:10.0\n"}
+	var loadResponseBody LoadResponseBody
+	if err := json.Unmarshal(b[:], &loadResponseBody); err != nil {
+		return "", err
+	}
+
+	return strings.TrimSpace(loadResponseBody.Stream)[14:], nil
```

Comment:
I have to parse the image name out of the CLI response message, which is awful, but thats the data we get back from docker.

I could use a regex, or a slice, however given this is unstructured data (see the example output a little further up), both are just as likely to break... When we update the docker client version, we may need to go update this :(